modules/ROOT/pages/afm-add-schedule.adoc

```diff
@@ -0,0 +1,42 @@
+= Add a Schedule
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+:imagesdir: ../assets/images
+
+You can add a schedule to an existing monitor if either of the following two conditions is true:
+
+* You plan to run the test associated with the schedule in a public location and the allocation of schedules for your organization in Anypoint Platform is not exhausted.
+* You plan to run the test associated with the schedule in a private location.
+
+When you add a schedule to a monitor, you can use the schedule to run any of the versions of the test for that monitor.
+
+== Procedure
+
+To add a schedule from the *Schedules* section of a monitor:
+
+. Select the monitor in which you want to add a schedule.
+. Click the *Schedules* tab.
+. Click *Add Schedule*.
+. In the Add Schedule dialog, specify this information:
++
+* The test version that you want the schedule to run
+* The location to run the test from
+* The frequency at which to run the test
+. Click *Add*.
```

Comment:
Once the form is visible, is this level of detail necessary? I'd rather see this space used to tell users how to fill out the form (reference to private locations, or a reference for how to write Cron).

Reply:
@runnabro , why would you like the doc to explain how to write cron jobs? The UI doesn't require users to do write them. Also, what private locations are is described in a separate topic. I'll add a link to that topic.